manager.py

```diff
@@ -52,7 +52,8 @@ def phases_permit_new_job(phases, d, sched_cfg, dir_cfg):
 
     milestone = ( sched_cfg['tmpdir_stagger_phase_major'],
                   sched_cfg['tmpdir_stagger_phase_minor'] )
-    if len([p for p in phases if p < milestone]) > 0:
+    max_allowed_before_milestone = sched_cfg['tmpdir_stagger_phase_limit']
```

Comment:
One effect of this is that people already running plotman with a customized config.yaml will start to crash when they pick up this code, until they add the new line to their config.  Should we make this config value explicitly optional?

What we really need is some more managed install and upgrade process.  @altendky FYI

Reply:
Nice catch! Updated the PR.